docs/en/api/cli.md

```diff
@@ -29,30 +29,76 @@ This command is pretty helpful when you need to have a quick overview of your ap
 <Command name="init" />
 
 ```
-  Description
-    Inits the Tauri template. If Tauri cannot find the src-tauri/tauri.conf.json
-    it will create one.
-  Usage
-    $ tauri init
-  Options
-    --help, -h        Displays this message
-    --force, -f       Force init to overwrite [conf|template|all]
-    --log, -l         Logging [boolean]
-    --directory, -d   Set target directory for init
-    --tauriPath, -t   Path of the Tauri project to use (relative to the cwd)
+Initializes a Tauri project
+
+USAGE:
+    cargo tauri init [FLAGS] [OPTIONS] [SUBCOMMAND]
+
+FLAGS:
+        --ci         Skip prompting for values
+    -f, --force      Force init to overwrite the src-tauri folder
+    -h, --help       Print help information
+    -l, --log        Enables logging
+    -V, --version    Print version information
+
+OPTIONS:
+    -A, --app-name <app-name>            Name of your Tauri application
+    -d, --directory <directory>          Set target directory for init
+    -D, --dist-dir <dist-dir>            Web assets location, relative to <project-dir>/src-tauri
+    -P, --dev-path <dev-path>            Url of your dev server
+    -t, --tauri-path <tauri-path>        Path of the Tauri project to use (relative to the cwd)
+    -W, --window-title <window-title>    Window title of your Tauri application
+
+SUBCOMMANDS:
+    help      Print this message or the help of the given subcommand(s)
+    plugin    Initialize a Tauri plugin.
+```
+
+### `init plugin`
+
+<Command name="init plugin" />
+
+```
+Initializes a Tauri plugin project.
+
+USAGE:
+    cargo tauri init plugin [FLAGS] [OPTIONS] --name <name>
+
+FLAGS:
+    -a, --api        Initializes a Tauri plugin with TypeScript API.
+    -h, --help       Print help information
+    -V, --version    Print version information
+
+OPTIONS:
+    -d, --directory <directory>      Set target directory for init
+    -n, --name <name>                Name of your Tauri plugin
+    -t, --tauri-path <tauri-path>    Path of the Tauri project to use (relative to the cwd)
```

Comment:
I forgot this one. Thought it was just _renamed_ to `plugin init`, but neither version was part of beta.8

Reply:
Ok so I removed the `init plugin` section, was that right?